src/domain/networking/packets/PacketScribe.ts

```diff
@@ -72,6 +73,8 @@ import BulkAvatarTraitsAck from "./BulkAvatarTraitsAck";
  *      {@link PacketScribe.AvatarIdentity&period;write|AvatarIdentity&period;write}
  *  @property {function} DomainConnectRequest.write -
  *      {@link PacketScribe.DomainConnectRequest&period;write|DomainConnectRequest&period;write}
+ *  @property {function} EntityData.read -
+ *      {@link PacketScribe.EntityData&period;read|EntityData&period;read}
```

Comment:
The link in the doc isn't working...
![image](https://user-images.githubusercontent.com/7455448/169413216-be2c5aed-68fc-488f-b2d3-fc85bb44408f.png)

Reply:
Fixed.